1657407 Add descriptions for SEO. (#17372)
dotnet/docs: docs/framework/install/run-net-framework-1-1-apps.md

---
title: "Run .NET Framework 1.1 apps on Windows 8, Windows 8.1, or Windows 10"

> description: "Describes how to accommodate apps that require the .NET Framework 1.1 which is no longer supported on many versions of the Windows operating system."

Reviewer: Is this better? Is this correct?

```suggestion
description: "Describes how to accommodate apps that require the .NET Framework 1.1 which is no longer supported by Microsoft."
```

> description: "Describes how to accommodate apps that require the .NET Framework 1.1 which is no longer supported on many versions of the Windows operating system."

Reviewer: Is this better? Is this correct?

```suggestion
description: "Describes how to accommodate apps that require the .NET Framework 1.1 which is no longer supported by Microsoft."
```

Author: I don't see any place in docs that says MS still supports 1.1 in any context. This is probably correct.

Reviewer 2: @richlander would probably know.

Author: If @richlander could weigh in -

If we're not sure that some version of windows still runs 1.1, I think we should stay with my original wording.
